templates/rapidsai-core/partials/patch.dockerfile.j2

```diff
@@ -1,10 +1,8 @@
 {# This partial is used to install patches to resolve known CVEs #}
 
-{# Patch for CVE-2020-8116 https://github.com/advisories/GHSA-ff7x-qrg7-qggm #}
-{# Patch for CVE-2020-15123 https://nvd.nist.gov/vuln/detail/CVE-2020-15123 #}
-RUN source activate rapids \
```

Comment:
Why was this removed? Isn't it still necessary?

Reply:
I removed it because, if I remember correctly it was related to the 22.08 release.  I can leave it in.